NAMESPACE

```diff
@@ -20,10 +20,13 @@ export(df_md)
 export(fit_mmrm)
 export(fit_single_optimizer)
 export(free_cores)
+export(kr)
 export(mmrm)
 export(mmrm_control)
 export(refit_multiple_optimizers)
+export(tr)
```

Comment:
would recommend to not export `tr` as this is internal helper, also would prefix with `h_`